### PR TITLE
Add a CHANGELOG

### PR DESCRIPTION
**Pull request by davidbeckingsale**

### CODE REVIEW

**Comment by davidbeckingsale** [comment]
@mcfadden8 - please check this against the current "done" JIRA items

File: CHANGELOG [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+and the smart pointers added unecessary overhead.
+
+### Removed
+
+- The 'coalesce' method was removed from ResourceManager and now must be
+accessed directory. examples/cookbook/recipe_coalesce_pool.cpp shows how to do
+this.
+
+### Fixed
+
```
Comment (mcfadden8) [inline CHANGELOG]:
```suggestion
+
+- The replay tool can now also replay captured release operations.
```

Comment (davidbeckingsale) [reply CHANGELOG]: Is this a fix or a change or an addition?

Comment (mcfadden8) [reply CHANGELOG]: It is a fix.  Replay was recording the release event, but the replay tool was not replaying it.

Comment (davidbeckingsale) [reply CHANGELOG]: Okay, makes sense. Maybe we should reword this item so it makes that clear. As someone who didn't know the background, it seems like a new feature.

Comment (mcfadden8) [reply CHANGELOG]: I updated my suggestion.  Thoughts?  (I wish we could collaborate on my updates)

**Pull request merged**